Sources/StreamChat/Workers/ChannelListCleanupUpdater.swift

```diff
@@ -0,0 +1,108 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+/// Cleans up local channels data and refetches it from backend
+class ChannelListCleanupUpdater<ExtraData: ExtraDataTypes>: Worker {
```

Comment:
I'd really appreaciate if you spend more time on the documentation here + on the methods of the class 👍

Reply:
Since this updater is touching `ChannelDTO` and it also reloads all existing queries, can we maybe call it something like `DatabaseCleanupUpdater`?